services/core-api/app/api/mines/project_summary/response_models.py

```diff
@@ -28,23 +29,13 @@
     'description': fields.String
 })
 
-PROJECT_SUMMARY_STATUS_CODE_MODEL = api.model('ProjectSummaryStatusCode', {
```

Comment:
was this model just dulicated?

Reply:
yes it appeared to be, caused some weird bugs with what was being returned to the FE.